.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish distribution to PyPI
+
+on:
+  release:
```

Comment:
Just to clarify, this is every time there's a new release in https://github.com/aws/amazon-braket-sdk-python/releases?

Reply:
Yep

Reply:
Thanks. How do we know it's successful? Just check the PyPI website, or is there a notification/logs we can view?

Reply:
Checking PyPi is the end-all-be-all, but there's also the repo's "Actions" tab that will give a history of every run. I think you can configure your account's notifications for Actions too.

Reply:
Is it possible to run integ/unit tests before the PyPI release? For example, instead of having it in Github Actions, and instead triggered by webhook in codebuild?

Reply:
In any normal circumstance, the unit/integ tests _will_ run before the PyPi release. When a new version comes in, tests are run, and a new release is only created if they all pass. It's certainly possible to put the PyPi release in that workflow; however, manual releases, if they're ever needed, will also require a corresponding manual publish to PyPi.

Reply:
There's a wait time for alarms after a commit has been merged to main for the unit/integ tests to run, right? It might be that the release is created before sufficient wait time for a problem to emerge from unit/integ tests.

I'm suggesting if we had a webhook trigger for releases, which ran the unit/integ tests and published to PyPI, so it would work for a manual release or when we have an automatic version updating, the automatic one.

Reply:
Are you talking about manual releases causing problems? In the automatic case, the release cannot be created unless the tests all pass.

Reply:
Yes, since we only have manual releases right now. Manual as in we have to do "Create a release" in the UI.

What do you mean by "automatic case"?

Basically the benefit of when there's a release, triggering the integ/unit tests to run, then publishing to PyPI.

Reply:
Also, perhaps this can help with running unit/integ tests: https://github.com/aws-actions/aws-codebuild-run-build

Reply:
Right, releases are in fact automatic; as soon as the version number is bumped, a release is automatically created. The main future change would be automatic version bumping.

Reply:
What about the release notes for an automatic update? Doesn't that have to be manually entered?

Reply:
Yeah, the way it currently stands, the release notes would have to be post-edited.

Reply:
OK, from the point of view of manual releases, I think once we have integ tests running as part of the PR builds we should be good.